ansible/cluster.yml

```diff
@@ -44,6 +44,12 @@
   tags:
     - network-service-install
 
+# install contiv netmaster
```

Comment:
Is there any way to move flannel, opencontrail and contiv into their own playbook(s)/file(s)? declutter the top level?